### DISCUSSION TITLE
OOM while generate a large file

### DISCUSSION THREAD

**Discussion by yanghaoustc** [qax-os/excelize#1214] [Q&A]
i need to support a large excel,but it will leads to OOM,i could use java and Apache POI by its sxssf to  support low memory consumption, and i want to know if excelize could also supports such operation？
can anyone help me plz!!!!

**Comment by xuri** [comment]
Which version of Excelize are you using, and how much data is in your spreadsheet, how much memory cost? Please using the [stream writer](https://xuri.me/excelize/zh-hans/stream.html#NewStreamWriter) to generate large spreadsheets.

**Reply by yanghaoustc** [reply]
it works! Thank you!
